nidaqmx/_task_modules/channel_collection.py

```diff
@@ -5,7 +5,11 @@
 
 import ctypes
 import six
-from collections import Sequence
+
+if six.PY2:
```

Comment:
We are removing support for python 2.7, so please simplify those imports to only:

`from collections.abc import Sequence`

Reply:
Thanks, the setup.py still had 2.7 classifier at https://github.com/ni/nidaqmx-python/blob/0058a029cca6e6115bac1a3fcdc0c46c3e5e60e9/setup.py#L64 due to which I added the check. I will use your suggestion